hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java

```diff
@@ -928,6 +928,8 @@ public void usage(int exitCode) {
       o.printf("--add-tags {...}       Comma-delimited. Adds these on repost.\n");
       o.printf("--remove-tags {...}    Comma-delimited. Removes these on repost.\n");
       o.printf("--confidence {...}\n");
+      o.printf("-s|--status {...}\n");
+      o.printf("-r|--review-status {...}\n");
       o.printf("--precision {...}\n");
```

Comment:
add print help for '--related-ids-for-upload' ?

Reply:
good call